<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigPowerMeter</class>
 <widget class="QMainWindow" name="ConfigPowerMeter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="instrumentInfoLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>771</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Instrument: </string>
    </property>
   </widget>
   <widget class="QLabel" name="instrumentAddressLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>771</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Address: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="testEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>680</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="testLabel">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>680</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Test</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="radioButtonGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>1001</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Select Channel/Slot</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>1001</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Adjust Power Settings</string>
    </property>
    <widget class="QLineEdit" name="powerDisplay">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>80</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="powerDisplayUnit">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>90</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>(unit)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>331</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Power Unit</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>331</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Power</string>
     </property>
    </widget>
    <widget class="QComboBox" name="powerUnitComboBox">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>30</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <item>
      <property name="text">
       <string>Watt</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>dBm</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>1001</width>
      <height>231</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Adjust Wavelength Settings</string>
    </property>
    <widget class="QLineEdit" name="wavelengthDisplay">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>80</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="minWavelengthDisplay">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>130</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="maxWavelengthDisplay">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>180</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="wavelengthDisplayUnitLabel">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>90</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>(unit)</string>
     </property>
    </widget>
    <widget class="QLabel" name="minWavelengthUnitLabel">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>140</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>(unit)</string>
     </property>
    </widget>
    <widget class="QLabel" name="maxWavelengthUnitLabel">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>190</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>(unit)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>331</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Wavelength Unit</string>
     </property>
    </widget>
    <widget class="QLabel" name="wavelengthDisplay_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>331</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Wavelength</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>331</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Min Wavelength</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>331</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Max Wavelength</string>
     </property>
    </widget>
    <widget class="QComboBox" name="wavelengthComboBox">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>30</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>m</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>mm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>um</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>nm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>pm</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="wavelengthEdit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>80</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="wavelengthEditUnitLabel">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>90</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>(unit)</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>10</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Configuration</string>
    </property>
    <widget class="QPushButton" name="saveSettingsButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>80</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loadSettingsButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>80</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
   </widget>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox</zorder>
   <zorder>instrumentInfoLabel</zorder>
   <zorder>instrumentAddressLabel</zorder>
   <zorder>testEdit</zorder>
   <zorder>testLabel</zorder>
   <zorder>radioButtonGroupBox</zorder>
   <zorder>groupBox_3</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
